protostuff-parser/src/main/antlr3/io/protostuff/parser/ProtoParser.g

```diff
@@ -113,6 +116,12 @@ annotation_keyval [Proto proto, Annotation annotation]
         )
     ;
 
+doc_entry [Proto proto]
+    :   DOC {
+            proto.addDoc($DOC.text.substring(3).trim());
```

Comment:
On Thu, Apr 16, 2015 at 8:20 PM, Kostiantyn Shchepanovskyi <
notifications@github.com> wrote:

> In protostuff-parser/src/main/antlr3/io/protostuff/parser/ProtoParser.g
> <https://github.com/protostuff/protostuff/pull/119#discussion_r28504346>:
>
> > @@ -113,6 +116,12 @@ annotation_keyval [Proto proto, Annotation annotation]
> >          )
> >      ;
> >
> > +doc_entry [Proto proto]
> > +    :   DOC {
> > +            proto.addDoc($DOC.text.substring(3).trim());
>
> You did not understand my point. I already have comments like this in my
> code:
>
> /// Message description
> /// Usage:
> /// <code>
> /// class X extends Foo<Bar> {
> ///     public static final int BAZ = 1;
> ///     int z() {
> ///         // ...
> ///     }
> /// }
> /// </code>
>
> This comment should be transformed to java comment:
>
> /**
>  * Message description
>  * Usage:
>  * <code>
>  * class X extends Foo<Bar> {
>  *     public static final int BAZ = 1;
>  *     int z() {
>  *         // ...
>  *     }
>  * }
>  * </code>
>  */
>
> Your solution is completely ok, the only thing I need is to remowe that
> .trim() call.
>
I see.  I guess we can remove the trim call.
Or we can just remove/strip the space(s) after the triple-slash
(prefix-trim)

> —
> Reply to this email directly or view it on GitHub
> <https://github.com/protostuff/protostuff/pull/119/files#r28504346>.
>



-- 
When the cat is away, the mouse is alone.
- David Yu